src/btree/bt_read.c

```diff
@@ -609,6 +613,10 @@ __wt_page_in_func(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags
                 continue;
         }
         __wt_spin_backoff(&yield_cnt, &sleep_usecs);
+        ++sleep_count;
+        if (sleep_count > 10 * WT_THOUSAND && sleep_count % 10 * WT_THOUSAND == 0)
+            __wt_verbose_warning(session, WT_VERB_READ,
+              "sleep to wait the page for %" WT_SIZET_FMT " times", sleep_count);
```

Comment:
How about adding the `ref` here to find out clearly to which page this log belongs to?